sample_apps/simple_app/cleanup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# ##############################################################################
+# Helper script to clean-up running processes after a successful / failed
```

Comment:
Initially I had tried to inline this cleanup / kill logic in `run_example.sh`. But any kill results in an error and so the outer script fails, causing CI to fail.

So, I had to painfully provide this separate cleanup-only script.